docs/reference/modules/node.asciidoc

```diff
@@ -1,32 +1,48 @@
 [[modules-node]]
 == Node
 
-*elasticsearch* allows to configure a node to either be allowed to store
-data locally or not. Storing data locally basically means that shards of
-different indices are allowed to be allocated on that node. By default,
-each node is considered to be a data node, and it can be turned off by
-setting `node.data` to `false`.
+*elasticsearch* allows the configuration of a node to play multiple
```

Comment:
*elasticsearch* nodes have three roles and any elasticsearch node can play one to three of those roles. By default every node performs all three roles.